com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/SurfaceSubTarget.cs

```diff
@@ -159,7 +159,7 @@ protected override void CollectPassKeywords(ref PassDescriptor pass)
             pass.keywords.Add(CoreKeywordDescriptors.TransparentWritesMotionVector);
             pass.keywords.Add(CoreKeywordDescriptors.FogOnTransparent);
 
-            if (pass.IsLightingOrMaterial())
+            if (pass.IsLightingOrMaterial() || pass.IsForwardEmissiveForDeferred())
```

Comment:
Does it make sense to decouple the debug display check from `LightingOrMaterial`? Not sure if this new emissive pass qualifies as "lighting or material" (and also, adding it into the LightingOrMaterial check causes other issues).

Reply:
Here I will suggest to make thing clearer. 

Let's create a
pass.NeedDebugDisplay()    (or any better name :) 

with 
public static bool NeedDebugDisplay(this PassDescriptor pass) =>
 return pass.IsLightingOrMaterial() || pass.lightMode == HDShaderPassNames.s_ForwardEmissiveForDeferredStr

At least the purpose of the function is clear.

> (and also, adding it into the LightingOrMaterial check causes other issues).
yes, it shouldn't be added to IsLightingOrMaterial, it is different.

Reply:
Done!